experiment/run_experiment.py

```diff
@@ -679,30 +682,30 @@ def main():
     fuzzers = args.fuzzers or all_fuzzers
 
     concurrent_builds = args.concurrent_builds
-    if concurrent_builds is not None:
-        if not concurrent_builds.isdigit():
-            parser.error(
-                'The concurrent build argument must be a positive number')
-        concurrent_builds = int(concurrent_builds)
+    if concurrent_builds is not None and concurrent_builds <= 0:
+        parser.error('The concurrent build argument must be a positive number,'
+                     f' received {concurrent_builds}.')
+
     runners_cpus = args.runners_cpus
-    if runners_cpus is not None:
-        if not runners_cpus.isdigit():
-            parser.error('The runners cpus argument must be a positive number')
-        runners_cpus = int(runners_cpus)
+    if runners_cpus is not None and runners_cpus <= 0:
+        parser.error('The runners cpus argument must be a positive number,'
+                     f' received {runners_cpus}.')
+
     measurers_cpus = args.measurers_cpus
-    if measurers_cpus is not None:
-        if not measurers_cpus.isdigit():
-            parser.error(
-                'The measurers cpus argument must be a positive number')
-        if runners_cpus is None:
-            parser.error(
-                'With the measurers cpus argument you need to specify the'
-                ' runners cpus argument too')
-        measurers_cpus = int(measurers_cpus)
+    if measurers_cpus is not None and measurers_cpus <= 0:
+        parser.error('The measurers cpus argument must be a positive number,'
+                     f' received {measurers_cpus}.')
+
+    if runners_cpus is None and measurers_cpus is not None:
+        parser.error('With the measurers cpus argument (received '
+                     f'{measurers_cpus}) you need to specify the runners cpus '
+                     'argument too.')
+
     if (runners_cpus if runners_cpus else 0) + (measurers_cpus if measurers_cpus
                                                 else 0) > os.cpu_count():
-        parser.error('The sum of runners and measurers cpus is greater than the'
-                     ' available cpu cores (%d)' % os.cpu_count())
```

Comment:
Here we verify if `runners_cpu + measurers_cpu < os.cpu_count()`.
I presume this implies that `runners` and `measurers` will be in the same machine even for cloud experiments?